readthedocs/oauth/tasks.py

```diff
@@ -158,7 +161,14 @@ def sync_active_users_remote_repositories():
         try:
             # NOTE: sync all the users/repositories in the same Celery process.
             # Do not trigger a new task per user.
-            sync_remote_repositories(user.pk)
+            # NOTE: We skip the GitHub App, since all the repositories
+            # and permissions are keep up to date via webhooks.
+            # Triggering a sync per-user, will re-sync the same installation
+            # multiple times.
+            sync_remote_repositories(
+                user.pk,
```

Comment:
We could probably have a wrapper around this function, so it accepts a user object, instead of re-fetching the object from the DB each time.